client/dashboard/sites/settings-callout/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { __experimentalText as Text, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { settings } from '@wordpress/icons';
+import { addQueryArgs } from '@wordpress/url';
+import { Callout } from '../../components/callout';
+import calloutIllustrationUrl from './callout-illustration.svg';
+import type { CalloutProps } from '../../components/callout/types';
+import type { ReactNode } from 'react';
+
+interface SettingsCalloutProps extends Omit< CalloutProps, 'title' | 'description' > {
+	siteSlug: string;
+	title?: string;
+	description?: ReactNode;
+}
+
+export default function SettingsCallout( {
+	siteSlug,
+	icon,
+	imageSrc,
```

Comment:
would `illustration` be a better name?

Reply:
We had the exact opposite comment from the DS team here 😄 https://github.com/Automattic/wp-calypso/pull/103414#discussion_r2095412011

Reply:
Follow-up in https://github.com/Automattic/wp-calypso/pull/103414#discussion_r2107948563.